contributors.jenkins.io.tf

```diff
@@ -0,0 +1,28 @@
+resource "azurerm_resource_group" "contributors_jenkins_io" {
+  name     = "contributors-jenkins-io"
+  location = var.location
+  tags     = local.default_tags
+}
+
+resource "azurerm_storage_account" "contributorsjenkinsio" {
+  name                      = "contributorsjenkinsio"
+  resource_group_name       = azurerm_resource_group.contributors_jenkins_io.name
+  location                  = azurerm_resource_group.contributors_jenkins_io.location
+  account_tier              = "Standard"
+  account_replication_type  = "GRS"
```

Comment:
What is the reasoning behind using `GRS` here? (vs. `LRS` or `ZRS`) . Could you explain it in the issue (and the eventual cost impact)